src/server/api/controllers/candidates.controller.js

```diff
@@ -35,11 +36,25 @@ const createCandidate = async (userId, boardId, newCandidate) => {
   if (typeof newCandidate.name !== 'string') {
     throw new IncorrectEntryError(`Candidate name should be string`);
   }
-  const createNewCandidate = await knex('candidates').insert({
+  await checkUserRole({ userId, boardId });
+  const createNewCandidate = await knex('candidates').returning('id').insert({
     boardId,
     name: newCandidate.name,
     isBlocked: false,
   });
+  const getAllMembersOfBoard = await getMembers.getAllMembers(userId, boardId);
+  const allMembersId = getAllMembersOfBoard.map((member) => member.userId);
```

Comment:
You may have to add the owner's user id to this array. (I'm not sure right now if the owner's id is part of the members.)

Reply:
HI Moritz, The owner id is part of the members list.